.env

```diff
@@ -0,0 +1,9 @@
+#
+# Cycle to build
+#
+cycle=c0014
+
+#
+# Development cycle to build
+#
+dev_cycle=c0021.007
```

Comment:
So, this worries me a bit. It is basically tying a particular version of the cycle to your development environment. 

My suggestion would be to do something similar to what we adopt on our packages managed via docker-compose, which is, create a `setup.env` directory elsewhere to set these values as environment variables. 

For instance, in `LOVE-integration-tools` you can have a `setup.env` file with `export dev_cycle=c0021.007` which you `source` before doing the build. This way, you will have only one place to update when you deploying your system, and you won't have to modify LOVE-producer to update to different cycles.